src/Text/Pandoc/Readers/ODT/ContentReader.hs

```diff
@@ -455,6 +474,12 @@ getListConstructor ListLevelStyle{..} =
     toListNumberDelim (Just "(") (Just ")") = TwoParens
     toListNumberDelim     _          _      = DefaultDelim
 
+--
+startingNumber :: Bool  -> Int -> Maybe ListLevelStyle -> Int
+startingNumber continueNumbering prevStartCounter mListLevelStyle
+    | continueNumbering                = prevStartCounter + 1
```

Comment:
Suppose we have a list
```
1. a
2. b
```
and then another list with "continue numbering."  The new list should begin with 3, but I think your code will make it begin with 2 (previous list's start number + 1).

Reply:
> The new list should begin with 3, but I think your code will make it begin with 2 (previous list's start number + 1).`

No it won't.

```hs
  where
    constructOrderedList startNum listLevel listItemCount =
          reader
      >>> modifyExtraState (shiftListLevel (-1))
      >>> modifyExtraState (modifyPreviousListStartCounter listLevel (startNum + listItemCount - 1))
      >>> arr (orderedListWith (startNum, DefaultStyle, DefaultDelim))
    constructListWith listLevelStyle startNum listLevel listItemCount =
          reader
      >>> getListConstructor listLevelStyle startNum
      ^>> modifyExtraState (shiftListLevel (-1))
      >>> modifyExtraState (modifyPreviousListStartCounter listLevel (startNum + listItemCount - 1))
```

The previousListStartCounter for that `listLevel` will be seto to `(startNum + listItemCount - 1)`. 

I need to give it better naming, as it is basically the last number on the list.

e.g. with

```md
1. a
2. b
3. c
```

the value will be set to 3.

I can probably refactor it to the value being set to `4`, renaming the counter to `listContinuationStartCounter` or something.